source/ember-inspector/container.md

```diff
@@ -0,0 +1,29 @@
+Ember.js implements dependency injection by using a container.  Every Ember
+application has a container that creates instances for you. You can
+inspect these instances using the inspector's container tab.
+
+
+<img src="/images/guides/ember-inspector/container-screenshot.png" width="680"/>
```

Comment:
images can't be absolute, since the output of a build lives inside a subdirectory.